packages/next/src/server/lib/router-utils/block-cross-site.ts

```diff
@@ -25,7 +26,7 @@ export const blockCrossSite = (
     }
     res.end('Unauthorized')
     warnOnce(
-      `Blocked cross-origin request to /_next/*. To allow this, configure "allowedDevOrigins" in next.config\nRead more: https://nextjs.org/docs/app/api-reference/config/next-config-js/allowedDevOrigins`
+      `Blocked cross-origin request to /_next/*. Cross-site requests are blocked in "no-cors" mode.`
```

Comment:
Tweaked this since I'm realizing it's the `sec-fetch-dest` case, which doesn't care about the `allowedDevOrigins` configuration.